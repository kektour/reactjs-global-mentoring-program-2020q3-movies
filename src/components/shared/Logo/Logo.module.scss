.root {
  margin: 0;
  color: #f65261;
  font-size: 24px;
  text-decoration: none;

  & > span {
    font-weight: bold;
  }

  &:hover {
    // cursor: pointer;
    cursor: default;
  }
}
