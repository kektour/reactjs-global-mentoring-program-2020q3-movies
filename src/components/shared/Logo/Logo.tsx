import React from 'react';
import classnames from 'classnames';
import styles from './Logo.module.scss';

type Props = {
  classes?: {
    root?: string;
  };
};

// TODO: Add link to /
export const Logo: React.FC<Props> = (props) => {
  const { classes = {} } = props;
  return (
    <p className={classnames(styles.root, classes.root)}>
      <span>netflix</span>roulette
    </p>
  );
};
