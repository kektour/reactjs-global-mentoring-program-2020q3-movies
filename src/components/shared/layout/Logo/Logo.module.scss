.root {
  margin: 0;
  color: #f65261;
  font-size: 24px;

  & > span {
    font-weight: bold;
  }
}
