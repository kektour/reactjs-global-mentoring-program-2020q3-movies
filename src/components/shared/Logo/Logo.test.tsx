import React from 'react';
import { render } from '@testing-library/react';
import { Logo } from './Logo';

describe('Logo', () => {
  it('Should render component correctly', () => {
    const component = render(<Logo />);
    expect(component.asFragment()).toMatchSnapshot();
  });
});
