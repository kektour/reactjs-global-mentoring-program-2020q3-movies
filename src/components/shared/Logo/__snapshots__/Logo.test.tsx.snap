// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Should render component correctly 1`] = `
<DocumentFragment>
  <p
    class="root"
  >
    <span>
      netflix
    </span>
    roulette
  </p>
</DocumentFragment>
`;
